konig-schemagen/src/main/java/io/konig/schemagen/java/JavaClassBuilder.java

```diff
@@ -62,6 +61,7 @@
 	private OwlReasoner reasoner;
 	private Graph graph;
 	private TypeInfo owlThing;
+	private ShapeHandler shapeHandler;

```

Comment:
This was part of the merging problem. With the merge of master, this file was reverted.